guess/touchscreen/action

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -eu
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+. guess-functions
+
+$TRACE_SOURCE "Processing ..."
+
+# The idea comes from tslib. Touchscreen devices have
+# the INPUT_PROP_DIRECT property, which is determined
+# by the first from zero bit of the property number
+
+INPUT_PROP_DIRECT=1
+bitmask=$((1 << $INPUT_PROP_DIRECT))
+
+for i in "$SYSFS_PATH"/class/input/event*/device/ ; do
```

Comment:
It'd be better to use
`for i in "$SYSFS_PATH"/class/input/event*/device/properties; then` instead

Reply:
I initially wondered why 'then' is better than 'do':)

Reply:
Whoops, my bad.